srcpkgs/pipewire/files/pipewire-pulse/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+mkdir -p /run/pulse
+chown pipewire:pipewire /run/pulse
```

Comment:
this could mess with system pulsaudio, in case someone has enabled both by mistake. It should probably be guarded

Reply:
This will need adding a check script to pulseaudio, right?

Reply:
actually it might be fine now that there's no chown anymore. the pipewire service will just fail, which is ok.

Reply:
It fails with "Address already in use", it's ok as is